src/Microsoft.AspNetCore.Authentication/ClaimsTransformationHandler.cs

```diff
@@ -27,8 +30,13 @@ public ClaimsTransformationHandler(IClaimsTransformer transform)
                 await PriorHandler.AuthenticateAsync(context);
                 if (_transform != null && context?.Principal != null)
                 {
+                    var transformationContext = new ClaimsTransformationContext
+                    {
+                        Context = _httpContext,
+                        Principal = context.Principal
+                    };
```

Comment:
Is it ok to create `transformationContext` object separately or should it be inlined into `TransformAsync` call?